pkg/cmd/auth.go

```diff
@@ -107,3 +172,21 @@ func newLogoutCommand() *cobra.Command {
 
 	return cmd
 }
+
+func writeAuthTokenConfig(cmd *cobra.Command, token string, tokenType config.AuthTokenType) {
+	if len(token) > 0 {
+		cfg := &config.Config{
+			ContextConfig: map[string]*config.ContextConfig{
+				Config.CurrentContext: {
+					Auth: &config.AuthConfig{
+						Tokens: map[config.AuthTokenType]string{
+							tokenType: token,
```

Comment:
Might want to `strings.TrimSpace` on this so people won't have to use `echo -n` (unless this is already automatically handled elsewhere).

Reply:
Ironically, I had originally added that logic but subsequently removed it. Having to use `-n` in some situations was annoying me as a user, but I felt like maybe I was just being unreasonable. =)